src/citations/index.ts

```diff
@@ -62,7 +64,20 @@ export const wuppertalStudie = new Citation({
   title: "CO2-neutral bis 2035: Eckpunkte eines deutschen Beitrags zur Einhaltung der 1,5-°C-Grenze",
   publisher: "Wuppertal Institut",
   date: "2020-10",
-  archiveUrl: "https://web.archive.org/https://epub.wupperinst.org/frontdoor/deliver/index/docId/7606/file/7606_CO2-neutral_2035.pdf",
+  archiveUrl:
+    "https://web.archive.org/web/20210916070821/https://epub.wupperinst.org/frontdoor/deliver/index/docId/7606/file/7606_CO2-neutral_2035.pdf",
+  comment: `Zusammengefasst in {cite(wuppertalStudieFactsheet)}.`,
```

Comment:
Sadly, a reference of `wuppertalStudieFactsheet` is not possilbe, here.
"Block-scoped variable 'wuppertalStudieFactsheet' used before its declaration."

Reply:
I left out the value of the `comment` in the constructor, but assigned it after defining the other citation.